ui/raidboss/data/02-arr/raid/t10.js

```diff
@@ -126,6 +128,8 @@
         'Spike Flail': 'Fléau à pointes',
         'Wild Charge': 'Charge sauvage',
       },
+      '~effectNames': {
+      },
```

Comment:
Erhöhte Verwundbarkeit
Erratischer Puls
Markiert
Stromschlag

Reply:
again, they are not used in either txt/js file so no need to include them.